app/src/main/java/tv/vizbee/demo/helper/SharedPreferenceHelper.kt

```diff
@@ -0,0 +1,35 @@
+package tv.vizbee.demo.helper
+
+import android.content.Context
+import android.content.SharedPreferences
+
+class SharedPreferenceHelper(context: Context) {
```

Comment:
Make this singleton to avoid multiple instances through out the app

Reply:
Done